packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -273,13 +286,13 @@ describe("LRO Rest Client", () => {
       const poller = await getLongRunningPoller(client, initialResponse, {
         intervalInMs: 0
       });
-
       const result = await poller.pollUntilDone();
       if (isUnexpected(result)) {
         const error = `Unexpected status code ${result.status}`;
         assert.fail(error);
-        throw new Error(error);
       }
+      // FIXME
+      assert.equal(poller.getOperationState().status, "failed");
```

Comment:
Considering the `result.body.properties?.provisioningState` is failed status so I assume the operation state is also `falied` not `succeeded`

Reply:
This test operates in the resource location mode where success or failure is determined solely based on the status code of the response and since it is 200 in this case then the operation should be considered succeeded.